src/js/components/sharedComponents/dropdowns/NewPicker.jsx

```diff
@@ -154,7 +154,20 @@ const NewPicker = ({
                             onClick: createOnClickFn(option.onClick)
                         }))
                         .map((option) => (
-                            <li key={uniqueId()} className={`filter__dropdown-list-item ${option?.classNames ? option.classNames : ''} ${option.name.trim() === selectedOption.trim() ? 'active' : ''}`}>
+                            // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
li needs the remaining accessibility attributes and we should be able to get rid of this comment to disable the accessibility rules.  looks like the "role" is missing